support/runner-script.py

```
@@ -93,12 +92,12 @@ def _load_registry(file: str) -> dict[str, list[str]]:
 	return ret
 
 
-def _object_gcs_path(name: str, hash: str) -> str:
-	return f'genvm_runners/{name}/{hash}.tar'
+def _object_gcs_path(name: str, hash: str, ext: str) -> str:
```

Ruff (0.16.1)

[error] Function argument `hash` is shadowing a Python builtin

(A002)